### PR TITLE
[TT-10221] Add extraEnvs to tyk-bootstrap and update its documentation

### PR DESCRIPTION
**Pull request by buraksekili**
## Description

This PR adds `extraEnvs` section to tyk-bootstrap and adds a documentation about new environment variables.

## Related Issue

https://tyktech.atlassian.net/browse/TT-10221

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] Documentation updates or improvements.

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [ ] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
     - [ ] I have manually updated the README(s)/documentation accordingly.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: components/tyk-bootstrap/README.md [hunk 1]
```diff
@@ -13,25 +13,55 @@ The chart provides little benefits by itself and should only be used in conjunct
...
 See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To get all configurable options with detailed comments:
 
-    helm show values tyk-helm/tyk-bootstrap > values-bootstrap.yaml --devel
+```bash
+helm show values tyk-helm/tyk-bootstrap > values-bootstrap.yaml --devel
+```
     
 You can update any value in your local values.yaml file and use `-f [filename]` flag to override default values during installation. Alternatively, you can use `--set` flag to set it in Tyk installation.
+
+### Environment Variables
```
Comment (komalsukhani) [inline components/tyk-bootstrap/README.md:45]: Instead of defining environment variables here again should we just link it to k8s-bootstrap?

Comment (buraksekili) [reply components/tyk-bootstrap/README.md:45]: I can 👍 we'll update accordingly

Comment (buraksekili) [reply components/tyk-bootstrap/README.md:45]: updated in https://github.com/TykTechnologies/tyk-charts/pull/158/commits/dc72c35a650db34c2a427642d10c776b5f32e739. also, i am making this PR to draft until we have a release on tyk-k8s-bootstrap. Then, I'll update the image tags here in this PR and make the PR ready for testing. What do your think?

Comment (komalsukhani) [reply components/tyk-bootstrap/README.md:45]: Sounds good.

**Review by komalsukhani — APPROVED** [review]

**Pull request merged**